sqladmin/application.py

```diff
@@ -13,9 +13,7 @@
 from starlette.staticfiles import StaticFiles
 from starlette.templating import Jinja2Templates
 
-if TYPE_CHECKING:
-    from sqladmin.models import ModelAdmin
```

Comment:
I have remove this check to be able to pass the test script successfully, and don't get any circular import errors, did you remember the case need this check ?

Reply:
Not really, how can it create circular import, it's weird, let me see.